spec/unit/widgets/button_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rspec'
+
+module LibyuiClient
+  module Widgets
+    RSpec.describe Button do
+      let(:widget_controller) { instance_double('widget_controller') }
+      let(:widget_filter) { { id: 'foo' } }
+      subject { Button.new(widget_controller, widget_filter) }
+      describe '#click' do
+        it 'has a request with action press' do
+          allow(widget_controller).to receive(:send_action) {}
+          expect(subject).to receive(:action).with({ action: 'press' })
+          subject.click
+        end
+      end
+      # end of $click
```

Comment:
Do we really need these comments? I see that they are also not consistent, sometimes added before `end`, sometimes after. I would remove all of them.

Reply:
all removed